MuonAnalysis/MuonAssociators/interface/PropagateToMuonSetup.h

```diff
@@ -0,0 +1,81 @@
+#ifndef MuonAnalysis_MuonAssociators_interface_PropagateToMuonSetup_h
+#define MuonAnalysis_MuonAssociators_interface_PropagateToMuonSetup_h
+//
+//
+
+/**
+  \class    PropagateToMuonSetup PropagateToMuonSetup.h "HLTriggerOffline/Muon/interface/PropagateToMuonSetup.h" 
+
+  \brief Propagate an object (usually a track) to the second (default) or first muon station.
+
+*/
+
+#include "DataFormats/Candidate/interface/Candidate.h"
```

Comment:
It looks like that most of these includes are not needed, certainly not in this header file: could you please clean them up?

Reply:
Done.